README.md

```diff
@@ -13,7 +13,7 @@ Flutter Code Editor is a multi-platform code editor supporting:
 - Themes,
 - And many other features.
 
-![Basic Example](https://raw.githubusercontent.com/akvelon/flutter-code-editor/main/example/images/factorial.png)
+![Basic example](example/images/main.gif)
```

Comment:
Read about images here:
https://medium.com/flutter-senior/my-rules-when-publishing-a-dart-package-22d07d6fb19f

Reply:
I changed reference to `main` branch so they will be visible after merge.